src/app/shared/search/search.component.html

```diff
@@ -0,0 +1,6 @@
+<div id="search-container">
+  <input #searchBox (keyup)="onType(searchBox.value)" id="search-bar" type="search" placeholder="חפש">
```

Comment:
Use translator service instead of hard coded text (placeholder)

Reply:
If the `onType` function simply assigns the text value into a local variable, this could be changed to ngModel. It works more efficiently and gives you more options.

Reply:
Added translation

Reply:
"onType" was put temporary there, the component isn't finished, it's a template that will be used once we integrate video & channel search.
"onType" might need a different implementation in the future.